olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1563883405/eclipse-che-preview-kubernetes.crd.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
File name is `olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1563883405/eclipse-che-preview-kubernetes.crd.yaml` ... with `1563883405` ... yet the timestamp changed in `olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/eclipse-che-preview-openshift.package.yaml` is to `1563883406`. Is that expected?

Reply:
For now `kubernetes` and `openshift` packages are built one after the other, so that the last nightly timestamp can differ between both.